### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by eeberhard**
Update top-level readme with new links

## Description

This quick fix updates the links that are visible on the top-level organization page with the .github README file.

## Review guidelines

Estimated Time of Review: 2 minutes

## Checklist before merging:

- [x] Confirm that the relevant changelog(s) are up-to-date in case of any user-facing changes

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
 ## AICA Technology
 
-### Repositories
+This the home of the AICA robotics framework!
```
Comment (domire8) [inline profile/README.md:3]: Is the missing 'is' intentional? I would still approve it :smile:

**Review by domire8 — APPROVED** [review]
Nice

File: profile/README.md (hunk 1 above)
Comment (eeberhard) [reply profile/README.md:3]: haha thanks for spotting that

Comment (eeberhard) [inline profile/README.md:3]:
```suggestion
This is the home of the AICA robotics framework 🦾 
```

**Pull request merged**